README.md

```diff
@@ -2,6 +2,11 @@
 
 This repository is here to help you bootstrap your own OpenFisca [extension](http://openfisca.org/doc/contribute/extensions.html) package.
 
+**You should NOT fork it but download a copy of it** and
```

Comment:
Explicit what we mean by `download`: add the download link and explicit that the downloaded item isn't a git repository but its code?

```suggestion
**You should NOT fork it but [download a copy](https://github.com/openfisca/extension-template/archive/master.zip) of its source code** and
```